analysis/webservice/algorithms/HofMoeller.py

```diff
@@ -168,7 +168,7 @@ def calc(self, computeOptions, **args):
                 result = done_queue.get()
                 try:
                     error_str = result['error']
-                    self.log.error(error_str)
```

Comment:
I asked myself before what is the best way to handle loggers in object. I used to have logger as global variable in the file but I has notice that log was often an attribute of the object. Do you have an opinion on that ?